api/user/index.ts

```diff
@@ -1,7 +1,11 @@
 import { getUserSessionTimeout, RoleGroupSessionTimeout } from '@hmcts/rpx-xui-node-lib'
+import { UserInfo } from 'auth/interfaces/UserInfo'
```

Comment:
Again, semi-colons.

Reply:
Agreed, its done randomly. I have change this now.